Apps/WebClient/src/NewClientApp/src/components/public/vaccine-card/VaccineCardView.vue

```diff
@@ -0,0 +1,638 @@
+                                    >
+                                    if you have any questions about this
+                                    collection.
+                                </span></v-card
+                            >
+                        </v-overlay>
+                    </HgButtonComponent>
+                    <div class="text-center">
+                        <v-row class="my-4 no-gutters d-flex align-center">
+                            <v-col>
+                                <hr />
+                            </v-col>
+                            <v-col cols="auto">
+                                <h3 class="h5 ma-0 px-4 text-muted">OR</h3>
+                            </v-col>
+                            <v-col>
+                                <hr />
+                            </v-col>
+                        </v-row>
+                        <p>Already a Health Gateway user?</p>
+                        <router-link to="/login">
+                            <HgButtonComponent
+                                aria-label="BC Services Card Login"
+                                data-testid="btnLogin"
+                                variant="primary"
+                                class="login-button"
+                                text="Log in with BC Services Card"
+                            >
+                                <span>Log in with BC Services Card</span>
+                            </HgButtonComponent>
+                        </router-link>
+                    </div>
+                </div>
+            </v-form>
+            <div class="mt-6 px-4 px-sm-12 py-6 bg-white">
+                <h3 class="mb-4">Help in other languages</h3>
+                <p>
+                    Talk to someone on the phone. Get support in 140+ languages,
+                    including:
+                </p>
+                <p>
+                    <span lang="zh">國粵語</span> |
+                    <span lang="pa">ਅਨੁਵਾਦ ਸਰਵਿਸਿਜ਼</span> |
+                    <span lang="ar">خدمات-ت-رج-م-ه؟</span> |
+                    <span lang="fr">Français</span> |
+                    <span lang="es">Español</span>
+                </p>
+                <p>
+                    <strong>
+                        Service is available every day: 7 am to 7 pm or 9 am to
+                        5 pm on holidays.
+                    </strong>
+                </p>
+                <div class="my-4">
+                    <HgButtonComponent
+                        variant="secondary"
+                        text="Call: 1‑833‑838‑2323 (Toll‑Free)"
+                        href="tel:711"
+                    />
+                </div>
+                <div class="my-4">
+                    <HgButtonComponent
+                        variant="secondary"
+                        text="Telephone for the Deaf: Dial 711"
+                        href="tel:+18338382323"
```

Comment:
```suggestion
                        text="Call: 1‑833‑838‑2323 (Toll‑Free)"
                        href="tel:+18338382323"
                    />
                </div>
                <div class="my-4">
                    <HgButtonComponent
                        variant="secondary"
                        text="Telephone for the Deaf: Dial 711"
                        href="tel:711"
```

These refs look swapped